### ISSUE TITLE
Home page | Accessibility (Level A) 1.1.1 Non-text Content - Text alternative is missing for the logo 

### ISSUE THREAD

**Issue by ntiney** [twg/wikiwash-qa#45]
Environment:
Windows 7 / Firefox

Steps to reproduce:

1. Navigate to the following webpage http://wikiwash.org/ 
2. Right click on one of the logo  
3. Verify that the "img" tag that represents the image contains an corresponding "alt" attribute.
4. Verify that the alt attribute is appropriate for the selected element. 
Expected result:
4. Appropriate alt attribute is used for the logo

Actual result:
5. Alt attribute is missing for the logo
Reference:
https://www.w3.org/TR/UNDERSTANDING-WCAG20/text-equiv-all.html#text-altdef
![ww_home_111_logo](https://cloud.githubusercontent.com/assets/14282568/18037027/62b107b0-6d48-11e6-88d8-b4cb5c75dead.jpg)
